src/constants/captions_en.js

```diff
@@ -59,6 +59,7 @@ export default {
     writtenby: 'by',
     toteacherinstruction: 'To Teacher Instructions',
     tolesson: 'To Lesson',
+    reset: 'Start over',
```

Comment:
Hva med "Clear check boxes"? Det høres litt voldsomt ut med "Start over", og man aner ikke hva det betyr. (Hvis mulig, kan man jo ha en tooltip som sier "Clear all check boxes for this lesson".